.github/workflows/love-build.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
+    - uses: leafo/gh-actions-lua@v9
+    - uses: leafo/gh-actions-luarocks@v4
+    - name: Install Lua Packages
+      run: luarocks install moonscript --dev
+    - name: Compile MoonScript
+      run: python .github/workflows/compile.py
+    # Build the applications
+    - uses: nhartland/love-build@master
+      with:
+       app_name: 'my_selene_game'
```

Comment:
I can look into whether it's possible to extract this app name from conf.lua later